ui-grid/ui-grid.d.ts

```diff
@@ -861,7 +861,7 @@ declare namespace uiGrid {
          * By default it returns the `$$hashKey` property if it exists. If it doesn't it uses gridUtil.nextUid()
          * to generate one
          */
-        rowIdentity?(): any;
+        rowIdentity?(row: IGridRowOf<TEntity>): any;
```

Comment:
Sorry for the confusion. Actually, this PR was for the `refresh` method, but I didn't know enough about Git and pushed this commit ([ab12eb7](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/8947/commits/ab12eb7927be2b54ccb8e8caa037a57967118441)) to the same branch.
Should I revert/drop this commit or will it be merged along with the other ([9346a92](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/8947/commits/9346a921e033dc4e30c2a81a6e2451f7811cc8c5))?